yarn.lock

```diff
@@ -3118,7 +3118,7 @@
     "@backstage/backend-common" "^0.23.3"
     "@backstage/backend-plugin-api" "^0.7.0"
     "@backstage/plugin-catalog-node" "^1.12.4"
-    "@janus-idp/cli" "1.13.1"
+    "@janus-idp/cli" "^1.13.1"
```

Comment:
this might surprise us with unexpected cli bumps like the ones that broke us in 1.28 and 1.29 -- is it OK to stay pinned to 1.13.1?

Reply:
This is moot.  @JessicaJHee will be opening a PR upstream to remove the cli dependency as that should be a devdependency only